tools/run_python.sh

```diff
@@ -9,21 +9,21 @@
 # To modify this file, edit `tools/run_python.py` and then run
```

Comment:
You would mind correcting the typo here while are you at it:  it should be `.sh`, not `.py` on this line.

Reply:
Done.